transaction-client/src/main/java/org/apache/omid/tso/client/MockTSOClient.java

```diff
@@ -54,13 +76,31 @@
                 return new ForwardingTSOFuture<>(f);
             }
 
-            boolean canCommit = true;
+            Set<Long> tableIDs = new HashSet<Long>();
             for (CellId c : cells) {
-                int index = Math.abs((int) (c.getCellId() % CONFLICT_MAP_SIZE));
-                if (conflictMap[index] >= transactionId) {
+                tableIDs.add(c.getTableId());
+            }
+
+            boolean canCommit = true;
```

Comment:
Please add minimal comments to this code. Also, the method is long. Suggest to split it into helper functions, i.e., two boolean methods hasConflictsWithFences(), hasConflictsWithCommittedTransactions()